web/src/title/loader.ts

```diff
@@ -46,19 +47,38 @@ export async function loadFromLocalStorage(key: string) {
 
 // Returns OSM input and the boundary polygon, either from a pre-hosted pbf
 // file or from Overpass.
-async function getOsmInput(gj: any): Promise<[ArrayBuffer, Feature<Polygon>]> {
-  if (gj.study_area_name) {
+async function getOsmInput(
+  gj: any,
+  isCnt: boolean,
+): Promise<[ArrayBuffer, Feature<Polygon>]> {
+  if (isCnt) {
```

Comment:
This branching is a little awkward. Before a first public launch, I'd like to rethink namespacing the study area boundaries. It's a bit premature now, because possibly for Scotland, there'll be pre-built binary files that have more than just OSM data baked in.